requirements/edx/github.txt

```diff
@@ -58,7 +58,7 @@ git+https://github.com/edx/edx-lint.git@v0.3.0#egg=edx_lint==0.3.0
 -e git+https://github.com/edx/edx-reverification-block.git@ned/upgrade-django-1.8#egg=edx-reverification-block
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
 git+https://github.com/edx/edx-organizations.git@822492f2840299b400fc6013403a355120c40468#egg=edx-organizations==0.1.7
-git+https://github.com/edx/edx-proctoring.git@django1.8-upgrade#egg=edx-proctoring==0.11.2
+git+https://github.com/edx/edx-proctoring.git@django1.8-upgrade#egg=edx-proctoring==0.11
```

Comment:
And you are right: it will install the updated version, no matter whether it's going up or down, my mistake.